Tinfoil-for-Twitter/src/main/java/com/mill_e/twitterwrapper/activity/BaseTwitterWebViewActivity.java

```diff
@@ -265,15 +321,17 @@ protected void restoreWebView(Bundle inState) {
      *               true if we should use a custom user agent for mobile devices,
      *               false if not.
      */
-    protected void setUserAgent(boolean force, boolean mobile) {
-        if (force && mobile) {
+    protected void setUserAgent(boolean force, boolean mobile, boolean facebookBasic) {
+        if (force && mobile && !facebookBasic) {
             if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.JELLY_BEAN_MR2) {
                 mWebSettings.setUserAgentString(USER_AGENT_MOBILE_OLD);
             } else {
                 mWebSettings.setUserAgentString(USER_AGENT_MOBILE);
             }
-        } else if (force && !mobile) {
+        } else if (force && !mobile && !facebookBasic) {
             mWebSettings.setUserAgentString(USER_AGENT_DESKTOP);
+        } else if (force && mobile && facebookBasic) {
+            mWebSettings.setUserAgentString(USER_AGENT_BASIC);
```

Comment:
Not that I can tell. Also I've just seen from the prefs screen for choose site that there are now 2 new strange facebook options. Damn.